### PR TITLE
Fix e2e test

### PR DESCRIPTION
**Pull request by alexandermushinskiy**

### CODE REVIEW

**Comment by AlexSugak** [comment]
@alexandermushinskiy why not run e2e tests on build?

**Comment by alexandermushinskiy** [comment]
I removed e2e tests from build process because I did not find for now how to fix the following error:
Chrome version must be >= 65.0.3325.0

**Comment by AlexSugak** [comment]
@alexandermushinskiy please create corresponding issue then.

**Comment by alexandermushinskiy** [comment]
@AlexSugak the appropriate issue has been created [Issue 67](https://github.com/skycoin/skycoin-web/issues/67)

File: e2e/send_sky.e2e-spec.ts [hunk 1]
```diff
@@ -41,27 +41,32 @@ describe('Send Sky', () => {
...
-    expect<any>(page.getValidWidthWrongAmount()).toBeFalsy();
-  });
+  // it('should select valid wallet', () => {
+  //   expect<any>(page.selectValidWallet()).toBeTruthy();
+  // });
 
-  it('should send  sky', () => {
-    expect<any>(page.getCanSend()).toBeTruthy();
-  });
+  // it('should not enter wrong amount', () => {
```
[the remark this answers is not in the document]
Comment (alexandermushinskiy) [reply e2e/send_sky.e2e-spec.ts]: @AlexSugak, it's already there, see lines 44-46

**Pull request merged**